concepticondata/conceptlists/Gallatin-1836-180.tsv

```diff
@@ -0,0 +1,181 @@
+Gallatin-1836-180-40	40	Blood	946	BLOOD
+Gallatin-1836-180-41	41	Town, Village	3609	TOWN OR VILLAGE
+Gallatin-1836-180-42	42	Chief	301	CHIEFTAIN
+Gallatin-1836-180-43	43	Warrior	3608	WARRIOR
+Gallatin-1836-180-44	44	Friend	1325	FRIEND
+Gallatin-1836-180-45	45	House, Hut		
+Gallatin-1836-180-46	46	Kettle	854	KETTLE
+Gallatin-1836-180-47	47	Arrow	977	ARROW
+Gallatin-1836-180-48	48	Bow	994	BOW
+Gallatin-1836-180-49	49	Axe, Hatchet		
+Gallatin-1836-180-50	50	Knife	1352	KNIFE
+Gallatin-1836-180-51	51	Canoe, Boat		
+Gallatin-1836-180-52	52	Indian Shoes	3607	MOCCASIN
+Gallatin-1836-180-53	53	Bread	1368	BREAD
+Gallatin-1836-180-54	54	Pipe, Calumet		
+Gallatin-1836-180-55	55	Tobacco	974	TOBACCO
+Gallatin-1836-180-56	56	Sky, Heaven		
+Gallatin-1836-180-57	57	Sun	1343	SUN
+Gallatin-1836-180-58	58	Moon	1313	MOON
+Gallatin-1836-180-59	59	Star	1430	STAR
+Gallatin-1836-180-60	60	Day	1225	DAY (NOT NIGHT)
+Gallatin-1836-180-61	61	Night	1233	NIGHT
+Gallatin-1836-180-62	62	Light	630	LIGHT (RADIATION)
+Gallatin-1836-180-63	63	Darkness	11	DARKNESS
+Gallatin-1836-180-64	64	Morning	1339	MORNING
+Gallatin-1836-180-65	65	Evening	1629	EVENING
+Gallatin-1836-180-66	66	Spring	1537	SPRINGTIME
+Gallatin-1836-180-67	67	Summer	1342	SUMMER
+Gallatin-1836-180-68	68	Autumn	1276	AUTUMN
+Gallatin-1836-180-69	69	Winter	1471	WINTER
+Gallatin-1836-180-70	70	Wind	960	WIND
+Gallatin-1836-180-71	71	Lightning	1654	LIGHTNING
+Gallatin-1836-180-72	72	Thunder	1150	THUNDER
+Gallatin-1836-180-73	73	Rain	658	RAIN (PRECIPITATION)
+Gallatin-1836-180-74	74	Snow	784	SNOW
+Gallatin-1836-180-75	75	Hail	609	HAIL
+Gallatin-1836-180-76	76	Fire	221	FIRE
+Gallatin-1836-180-77	77	Water	948	WATER
+Gallatin-1836-180-78	78	Ice	617	ICE
+Gallatin-1836-180-79	79	Earth, Land	3603	EARTH OR LAND
+Gallatin-1836-180-80	80	Sea	1474	SEA
+Gallatin-1836-180-81	81	River	666	RIVER
+Gallatin-1836-180-82	82	Lake	624	LAKE
+Gallatin-1836-180-83	83	Valley	921	VALLEY
+Gallatin-1836-180-84	84	Hill	614	HILL
+Gallatin-1836-180-85	85	Mountain	639	MOUNTAIN
+Gallatin-1836-180-86	86	Island	622	ISLAND
+Gallatin-1836-180-87	87	Stone, Rock	2125	STONE OR ROCK
+Gallatin-1836-180-88	88	Copper	1825	COPPER
+Gallatin-1836-180-89	89	Iron	621	IRON
+Gallatin-1836-180-90	90	Maize	506	MAIZE
+Gallatin-1836-180-91	91	Tree	906	TREE
+Gallatin-1836-180-92	92	Wood	1803	WOOD
+Gallatin-1836-180-93	93	Leaf	628	LEAF
+Gallatin-1836-180-94	94	Bark	1204	BARK
+Gallatin-1836-180-95	95	Grass	606	GRASS
+Gallatin-1836-180-96	96	Oak	644	OAK
+Gallatin-1836-180-97	97	Pine-tree	1740	PINE
+Gallatin-1836-180-98	98	Flesh, Meat	2615	FLESH OR MEAT
+Gallatin-1836-180-99	99	Beaver	1194	BEAVER
+Gallatin-1836-180-100	100	Deer	1936	DEER
+Gallatin-1836-180-101	101	Bison, Buffalo	3142	BISON
+Gallatin-1836-180-102	102	Bear	1820	BEAR
+Gallatin-1836-180-103	103	Wolf	522	WOLF
+Gallatin-1836-180-104	104	Dog	2009	DOG
+Gallatin-1836-180-105	105	Fox	1312	FOX
+Gallatin-1836-180-106	106	Squirrel	1628	SQUIRREL
+Gallatin-1836-180-107	107	Rabbirt, Hark	2345	LEPORID (RABBIT OR HARE)
```

Comment:
What's up with these typos?

Reply:
My bad.

Reply:
Ah, thought it might be in the original, especially since there were two typos (of which you corrected only one).